### PR TITLE
Feaure/roles badges redesign

### PR DESCRIPTION
**Pull request by ice-eros**

### CODE REVIEW

**Review by robertpreoteasa — DISMISSED** [review]

**Review by ice-ares — APPROVED** [review]

**Review by ice-ares — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

File: src/components/SegmentedControl/index.tsx [hunk 1]
```diff
@@ -25,9 +26,10 @@ import Animated, {
   useSharedValue,
   withSpring,
 } from 'react-native-reanimated';
+import {rem} from 'rn-units';
 
 const DEFAULT_MARGIN = 8;
-const CONTROL_HEIGHT = 55;
+const CONTROL_HEIGHT = 58;
```
Comment (ice-orion) [inline src/components/SegmentedControl/index.tsx:32]: rem?

Comment (ice-orion) [inline src/components/SegmentedControl/index.tsx:32]:
```suggestion
const CONTROL_HEIGHT = rem(58);
```

File: src/components/SegmentedControl/index.tsx [hunk 2]
```diff
@@ -25,9 +26,10 @@ import Animated, {
   useSharedValue,
   withSpring,
 } from 'react-native-reanimated';
+import {rem} from 'rn-units';
 
 const DEFAULT_MARGIN = 8;
```
Comment (ice-orion) [inline src/components/SegmentedControl/index.tsx:31]:
```suggestion
const DEFAULT_MARGIN = rem(8);
```

File: src/components/SegmentedControl/index.tsx (hunk 1 above)
Comment (ice-orion) [inline src/components/SegmentedControl/index.tsx:32]:
```suggestion
const CONTROL_HEIGHT = rem(58);
```

File: src/components/SegmentedControl/index.tsx (hunk 2 above)
Comment (ice-eros) [reply src/components/SegmentedControl/index.tsx]: done

File: src/components/SegmentedControl/index.tsx (hunk 1 above)
Comment (ice-eros) [reply src/components/SegmentedControl/index.tsx]: done

Comment (ice-eros) [reply src/components/SegmentedControl/index.tsx]: done

**Review by ice-orion — APPROVED** [review]

**Review by ice-hector — APPROVED** [review]

**Review by robertpreoteasa — APPROVED** [review]

**Pull request merged**